tests/Doctrine/Tests/Common/Proxy/ProxyGeneratorTest.php

```diff
@@ -525,6 +526,72 @@ public function testPhp81NeverType()
         );
     }
 
+    /**
+     * @requires PHP >= 8.1.0
+     */
+    public function testPhp81ReadonlyPublicProperties()
+    {
+        $className = Php81ReadonlyPublicPropertyType::class;
+        $proxyClassName = 'Doctrine\Tests\Common\ProxyProxy\__CG__\\' . $className;
+        $initializationData = [
+            'id' => 'c0b5cb93-f01b-43f8-bc66-bc943b1ebcfd',
+            'readable' => 'This field is read-only.',
+            'writeable' => 'This field is writeable.',
+        ];
+
+        if ( ! class_exists($proxyClassName, false)) {
+            $metadata = $this->createClassMetadata($className, ['id']);
+
+            $metadata
+                ->expects($this->any())
```

Comment:
I remember already commenting on this, but for some reason my comment does not appear… maybe I discarded by accident. 

You can drop this line, it's equivalent.
```suggestion
```